doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
+
+## Unlocking Account and Connecting to Ethereum
+
+Unlocking your Ethereum account and connecting to a JSON-RPC provider are necessary steps to run the Livepeer node unless it's in standalone transcoder mode. 
+
+For brevity we'll exclude these flags in the examples by `<...ETH SETUP...>` but these should always be required. If no Ethereum account is available the Livepeer node will create one for you.
+
+```
+livepeer
+    -network <mainnet|rinkeby> \
+    -ethUrl <JSON_RPC_PROVIDER> \
+    -ethKeystorePath <PATH_TO_KEY> \
```

Comment:
`-ethKeystorePath` points to a directory and not to the keystore file itself so this should be updated to something like `<PATH_TO_KEYSTORE_DIR>`.

Reply:
fixed in 4131a02b